src/cell.rs

```diff
@@ -1,70 +1,64 @@
+use crate::utils::{Multipolygon, Point};
 use std::collections::HashSet;
 
-use multipolygon::Point;
-use multipolygon::Multipolygon;
-
-pub mod multipolygon;
-
-#[derive(Debug, Default, Clone, PartialEq, Eq)]
 /// One cell in the country boundaries grid
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
 pub struct Cell {
     /// Areas that completely cover this cell
     pub containing_ids: Vec<String>,
     /// Id + Areas that only partly cover this cell
-    pub intersecting_areas: Vec<(String, Multipolygon)>
+    pub intersecting_areas: Vec<(String, Multipolygon)>,
 }
 
 impl Cell {
+    pub fn new(
+        containing_ids: Vec<String>,
+        intersecting_areas: Vec<(String, Multipolygon)>,
+    ) -> Self {
+        Self {
+            containing_ids,
+            intersecting_areas,
+        }
+    }
```

Comment:
Why? Just for convenience in the tests?

Reply:
mostly yes. If you don't want to expose Cell to the users, might not be needed.